### PR TITLE
REST events API prices

### PR DESCRIPTION
**Pull request by alessandrodalbello**
This PR is for adding prices model to REST API.

### CODE REVIEW

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/events/RunnersRequest.java [hunk 1]
```diff
@@ -43,4 +44,12 @@ public void setIncludeWithdrawn(boolean includeWithdrawn) {
         this.includeWithdrawn = includeWithdrawn;
     }
 
+    public boolean isIncludePrices() {
+        return includePrices;
+    }
+
+    public void setIncludePrices(boolean includePrices) {
```
Comment (volkodavs) [inline matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/events/RunnersRequest.java]: what about to use builder pattern for requests?

**Comment by alessandrodalbello** [comment]
Hold on, I've just noticed the copyright headers. Let me double check my IDE settings and get rid of them before merging.

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/Currency.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019 Triplebet Limited. All right reserved. Inchalla, Le Val, Alderney, GY9 3UL.
+ * Company Registration Number: 1827.
+ */
+
+package com.matchbook.sdk.core.clients.rest.dtos.prices;
+
+public enum Currency {
+
+    EUR, GBP, USD
```
Comment (volkodavs) [inline matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/Currency.java]: are you sure MB support only those currencies?

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/OddsType.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019 Triplebet Limited. All right reserved. Inchalla, Le Val, Alderney, GY9 3UL.
+ * Company Registration Number: 1827.
+ */
+
+package com.matchbook.sdk.core.clients.rest.dtos.prices;
+
+public enum OddsType {
+
+    DECIMAL, US
```
Comment (volkodavs) [inline matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/OddsType.java]: i think it should be more than those ones

**Comment by alessandrodalbello** [comment]
Commit amended. All good now.

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/events/RunnersRequest.java (hunk 1 above)
Comment (alessandrodalbello) [reply matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/events/RunnersRequest.java]: Agreed. I'll use the builder pattern for every request :+1:

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/Currency.java (hunk 2 above)
Comment (alessandrodalbello) [reply matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/Currency.java]: No, there are more. Though, as other options, I'm keeping the API as much simple as possible, at least initially. Even if it is probably safe to add all currencies from the beginning.

File: matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/OddsType.java (hunk 3 above)
Comment (alessandrodalbello) [reply matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/prices/OddsType.java]: Same as currencies...

**Pull request merged**